src/main/java/net/dv8tion/jda/internal/requests/restaction/ChannelActionImpl.java

```diff
@@ -146,7 +146,9 @@ public ChannelType getType()
     @CheckReturnValue
     public ChannelActionImpl<T> setSlowmode(int slowmode)
     {
-        Checks.check(slowmode <= 21600 && slowmode >= 0, "Slowmode must be between 0 and 21600 (seconds)!");
+        if (type != ChannelType.TEXT)
+            throw new IllegalStateException("Can only set slowmode on text channels");
```

Comment:
Is this exception documented anywhere?

Reply:
Here https://ci.dv8tion.net/job/JDA/javadoc/net/dv8tion/jda/api/requests/restaction/ChannelAction.html#setSlowmode(int%29